index.js

```diff
@@ -14,6 +14,8 @@ async function runAction() {
   const dockerfile = core.getInput('dockerfile') || '.';
   const registryHost = core.getInput('humanitec-registry') || 'registry.humanitec.io';
   const apiHost = core.getInput('humanitec-api') || 'api.humanitec.io';
+  const tag = core.getInput('tag') || null;
+  const autoTag = core.getInput('auto-tag') || false;
```

Comment:
As getInput returns a string, I would suggest this instead:
```
const autoTag = /^\s*(true|1)\s*$/i.test(core.getInput('auto-tag'));
```

This will return true for the following:
- `true`
- `TRUE`
- `True`
- `TrUE`
- `1`
And false for everything else